### PR TITLE
Added error codes/description to document

### PR DESCRIPTION
**Pull request by mariahhart**

### CODE REVIEW

File: api-reference/beta/api/notifications-post.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ localization_priority: Normal
 ms.prod: "notifications"
 doc_type: apiPageType
 author: "merzink"
+updates made by: "mariahhart"
```
Comment (Lauragra) [inline api-reference/beta/api/notifications-post.md]: Please don't add new attributes to the YAML headers; we include only the attributes defined in this topic: https://msgo.azurewebsites.net/add/document/guidelines/metadata.html.

File: api-reference/beta/api/notifications-post.md [hunk 2]
```diff
@@ -70,7 +71,7 @@ Content-type: application/json
 	"ios",
 	"android"
         ]
-    },
+ Not supported.    },
```
Comment (Lauragra) [inline api-reference/beta/api/notifications-post.md]: This doesn't look right - what is your intention here?

File: api-reference/beta/api/notifications-post.md [hunk 3]
```diff
@@ -46,6 +47,23 @@ In the request body, supply a JSON representation of a [notification](../resourc
 ## Response
 If successful, this method returns a `201 Created` response code that indicates that the notification was successfully created and stored. The notification will be subsequently fanned-out to all specified endpoints with a valid subscription. 
 
+The following table lists the possible error and response codes that can be returned.
```
Comment (Lauragra) [inline api-reference/beta/api/notifications-post.md]: FYI, we don't add custom H2 sections to API topics. I moved the error codes table into the Response section. I made a few minor edits to the descriptions here.

Comment (mariahhart) [reply api-reference/beta/api/notifications-post.md]: Thanks for the note, sorry about that.

File: api-reference/beta/api/notifications-post.md (hunk 1 above)
Comment (mariahhart) [reply api-reference/beta/api/notifications-post.md]: Got it.

File: api-reference/beta/api/notifications-post.md (hunk 2 above)
Comment (mariahhart) [reply api-reference/beta/api/notifications-post.md]: That must've been added by accident. I'll update it.

**Pull request merged**